crates/typst-library/src/text/raw.rs

```diff
@@ -199,6 +248,30 @@ impl Show for RawElem {
             }
 
             Content::sequence(seq)
+        } else if let Some(token) = lang {
+            let syntax_set =
+                load(&self.syntaxes(styles), &self.data(styles)).at(self.span())?;
+            if let Some(syntax) = syntax_set.find_syntax_by_token(&token) {
```

Comment:
can we get rid of that duplication by integrating the lazy loading into the else if above?